packages/cli/src/cliEntry.js

```diff
@@ -170,6 +169,9 @@ async function setupAndRun() {
   logger.setVerbose(commander.verbose);
 }
 
+// @todo replace this
+const findPlugins = () => {};
```

Comment:
you'll need to migrate RN itself to not use it for 0.60 anyway: https://github.com/facebook/react-native/blob/a2285b17907cfd3d7cfd311e18027da800a48b2e/jest/hasteImpl.js#L27

so feel free to remove it

Reply:
I was thinking about it but didn't have time to look into how React Native is using it yet. It would be great to get rid of it entirely and maybe just read config (`loadConfig` sounds like a better public API than `findPlugins`)